addOns/graphql/src/test/java/org/zaproxy/addon/graphql/VariantGraphQIUnitTest.java

```diff
@@ -127,6 +129,29 @@ void shouldLogWarnOnInvalidJson() throws HttpMalformedHeaderException {
         assertThat(logMessages.size(), is(1));
     }
 
+    @Test
+    void shouldSetGraphqlQueryParametersCorrectlyOnTheUrl()
+            throws HttpMalformedHeaderException, URIException {
+        // Given
+        HttpRequestHeader httpReqHeader = new HttpRequestHeader();
+        httpReqHeader.setMessage(
+                "GET /graphql/?query=%7BsqlInjection(expression:%20%221%22)%7D HTTP/1.1");
+        HttpMessage msg = new HttpMessage(httpReqHeader);
+
+        // When
+        variant.setMessage(msg);
+        NameValuePair param = variant.getParamList().get(0);
+        String sqliPaylaod = "\"or 1=1--";
```

Comment:
Better to test the escape in `InlineInjectorUnitTest` rather than here.

Reply:
Yes, added an extra test to the InlineInjectorUnitTest